src/components/tiltak/LagretTiltak.tsx

```diff
@@ -183,28 +178,29 @@ export const LagretTiltak = ({
             />
           )}
 
-          {!displayNyKommentar && !editererTiltak && (
+          {!displayNyKommentar && !editererTiltak && !vurdererTiltak && (
```

Comment:
Kanskje litt hårete med trippel not-sjekk, hadde det vært mulig å ha en enkel variabel som sier om den skal vises?

Reply:
Kan trekke det ut men trippel not-sjekk må leve et sted i en eller annen form :P